scalding-db/src/main/scala/com/twitter/scalding/db/macros/impl/ColumnDefinitionProviderImpl.scala

```diff
@@ -191,8 +191,10 @@ object ColumnDefinitionProviderImpl {
       case (cf: ColumnFormat[_], pos: Int) =>
         val fieldName = cf.fieldName.toStr
         val typeNameTerm = newTermName(c.fresh(s"colTypeName_$pos"))
+        // MySQL uses names like `DATE`, `INTEGER` and `VARCHAR`;
+        // Vertica uses names like `Date`, `Integer` and `Varchar`
         val typeName = q"""
-        val $typeNameTerm = $rsmdTerm.getColumnTypeName(${pos + 1})
+        val $typeNameTerm = $rsmdTerm.getColumnTypeName(${pos + 1}).toUpperCase(java.util.Locale.US)
```

Comment:
I'm confused by this. Can you comment how this works for Vertica if it expects non upper cased?